gammapy/catalog/hawc.py

```diff
@@ -67,29 +64,25 @@ def _info_position(self):
             f"Position error: {self.data.pos_err:.3f}\n"
         )
 
-    @staticmethod
-    def _info_spectrum_one(d, idx):
-        ss = f"Spectrum {idx}:\n"
-        val, err = d[f"spec{idx}_dnde"].value, d[f"spec{idx}_dnde_err"].value
-        ss += f"Flux at 7 TeV: {val:.3} +- {err:.3} cm-2 s-1 TeV-1\n"
-        val, err = d[f"spec{idx}_index"], d[f"spec{idx}_index_err"]
-        ss += f"Spectral index: {val:.3f} +- {err:.3f}\n"
-        radius = d[f"spec{idx}_radius"]
-        ss += f"Test Radius: {radius:1}\n\n"
-        return ss
-
     def _info_spectrum(self):
         """Print spectral info."""
         ss = "\n*** Spectral info ***\n\n"
-        ss += self._info_spectrum_one(self.data, 0)
+        ss += self._info_spectrum_one(0)
 
         if self.n_models == 2:
-            ss += self._info_spectrum_one(self.data, 1)
+            ss += self._info_spectrum_one(1)
```

Comment:
For the 3HWC, the info is a bit confusing.
There is a line "No second spectrum available for this source", which seems to imply that for some sources there is a 2nd one available, and then there is none. Perhaps this should not lie in the base class, but the `if` condition should be specific to the 2HWC? What do you think?

Reply:
or I remove  "for this source" then the statement is ok for both catalogues

Reply:
Yes, fine by me now.